quic-forward-proxy/src/inbound/proxy_listener.rs

```diff
@@ -49,17 +44,9 @@ impl ProxyListener {
                 .await;
 
         while let Some(connecting) = endpoint.accept().await {
-            let exit_signal = exit_signal.clone();
             let forwarder_channel_copy = forwarder_channel.clone();
             tokio::spawn(async move {
-                let connection = connecting.await.context("handshake").unwrap();
-                match Self::accept_client_connection(
-                    connection,
-                    forwarder_channel_copy,
-                    exit_signal,
-                )
-                .await
-                {
+                match Self::accept_client_connection(connecting, forwarder_channel_copy).await {
                     Ok(()) => {
                         debug!("connection handles correctly");
```

Comment:
Is is normal that after accepting connection we do not do anything?

Reply:
code is IMO okey, but naming is bad - will change _accept_ to _handle_